README.md

```diff
@@ -1,53 +1,67 @@
 # Description
-This module allows you to execute OQL queries from a microflow.It allows setting named parameters which will be properly escaped.
+This module allows you to execute OQL queries from a microflow. It allows setting named parameters which will be properly escaped.
 
 # Typical usage scenario
 Query data from the Mendix database and the result will be mapped to a Mendix entity.
```

Comment:
I'd add line breaks around this line and start this something like `The most typical scenario is ...`